### DISCUSSION TITLE
notes/developers/roadmap

### DISCUSSION THREAD

**Discussion by giscus** [v6d-io/v6d#1518] [General]
# notes/developers/roadmap

Vineyard aims to serve as an open-source in-memory immutable data manager. We cut a major release once a year, a minor release for about every two months, and a patch release every one or two weeks...

https://v6d.io/notes/developers/roadmap.html

**Comment by mgttt** [comment]
客户端能考虑支持原生windows吗？

**Reply by sighingnow** [reply]
可以，我加到这个月release的TODO 里面～

**Reply by sighingnow** [reply]
不过鉴于etcd-cpp-apiv3的依赖，可能只能支持msvc build了，mingw估计比较难。
